packages/react/src/components/Masthead/__tests__/MastheadTopNav.test.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright IBM Corp. 2016, 2018
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { settings as ddsSettings } from '@carbon/ibmdotcom-utilities';
+import MastheadTopNav from '../MastheadTopNav';
+import mockData from './data/response.json';
+import { mount } from 'enzyme';
+import React from 'react';
+import { TranslationAPI } from '@carbon/ibmdotcom-services';
+
+const { stablePrefix } = ddsSettings;
+
+jest.mock('@carbon/ibmdotcom-services', () => ({
+  TranslationAPI: {
+    getTranslation: jest.fn(() => Promise.resolve(mockData)),
+  },
+}));
+
+describe('MastheadTopNav', () => {
+  it('renders one MastheadTopNav', () => {
+    const wrapper = mount(<MastheadTopNav navigation={[]} />);
+
+    expect(wrapper).toHaveLength(1);
+  });
+
+  it('renders all the itens based in the `navigation` prop', async () => {
+    const data = await TranslationAPI.getTranslation();
```

Comment:
sorry looking at this again, I don't think we need to make this call in this test

you can probably just pass the mock data right into the `navigation` prop?

const wrapper = mount(<MastheadTopNav navigation={mockdata.links} />);

Reply:
no worries! You're absolutely right! Thanks, Anna! 👍